pkg/worker/mount.go

```diff
@@ -21,7 +23,21 @@ func NewContainerMountManager() *ContainerMountManager {
 
 // SetupContainerMounts initializes any external storage for a container
 func (c *ContainerMountManager) SetupContainerMounts(request *types.ContainerRequest) error {
+	// Create local workspace path so we can symlink volumes before the container starts
+	os.MkdirAll(defaultContainerDirectory, os.FileMode(0755))
+
 	for i, m := range request.Mounts {
+		if m.MountPath == defaultContainerDirectory && request.Stub.Type.IsDeployment() {
```

Comment:
we might want to check if it's **not** a serve instead. That way we can handle function runs as well